src/App.jsx

```diff
@@ -14,70 +15,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood === 'No goods selected'
+          ? 'No goods selected'
+          : `${selectedGood} is selected`}
+        {selectedGood !== 'No goods selected' && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              setSelectedGood('No goods selected');
+            }}
+          />
+        )}
+      </h1>
+      <table className="table">
+        <tbody>
+          {goods.map((good, index) => (
+            <tr
+              data-cy="Good"
+              key={good}
+              className={good === selectedGood
+                ? 'has-background-success-light'
+                : ''}
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  data-cy={good === selectedGood
```

Comment:
you do this check many times, I would recommend to save it in const : 
`const isGoodSelected = good === selectedGood;`